server/datastore/mysql/migrations/tables/20240202172022_CreateTableHostLockWipe.go

```diff
@@ -0,0 +1,32 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240202172022, Down_20240202172022)
+}
+
+func Up_20240202172022(tx *sql.Tx) error {
+	stmt := `
+		CREATE TABLE host_mdm_actions (
+			host_id INT UNSIGNED NOT NULL,
+			lock_ref VARCHAR(36) NULL,
+			wipe_ref VARCHAR(36) NULL,
+			suspended TINYINT(1) NOT NULL DEFAULT FALSE,
+			PRIMARY KEY (host_id)
```

Comment:
Secondary tables that reference a host must be added to [this list of tables](https://github.com/fleetdm/fleet/blob/main/server/datastore/mysql/hosts.go#L471) so that the entry is properly deleted when the host is deleted (because we can't have a FK on `hosts` for performance reason, we have to clean-up dependent tables manually). I think this should be part of the schema PR as that's where it makes the most sense. Note that you'll need to add an entry in this table for the `DeleteHost` test (there's a test that generates dependent records in all host-related tables and ensures that deleting the host deletes all dependent records: https://github.com/fleetdm/fleet/blob/main/server/datastore/mysql/hosts_test.go#L6192).

Reply:
Ok great good to know! I'll add it in here.